mscore/editstyle.cpp

```diff
@@ -780,24 +780,57 @@ EditStyle::EditStyle(Score* s, QWidget* parent)
       MuseScore::restoreGeometry(this);
       }
 
+void MuseScore::showStyleDialog(Element* e)
+      {
+      if (!_styleDlg)
+            _styleDlg = new EditStyle { cs, this };
+      else
+            _styleDlg->setScore(cs);
+      
+      if (e)
+            _styleDlg->gotoElement(e);
+      
+      if (_styleDlg->isVisible()) {
+            _styleDlg->raise();
+            _styleDlg->activateWindow();
+            }
+      else
+            _styleDlg->show();
+      }
+
+void MuseScore::showStyleDialogHeaderFooter()
```

Comment:
Since this function is identical to the one above except for one line, why not just check if `e` is a header or footer in `showStyleDialog()`?

Reply:
Good point. I felt indeed a little bad about this, but the problem was that there is no separate ElementType for Header/Footer. Your comment encouraged me to do a little more research, and I found a solution that is pretty okay. So thank you! 😊